config/locales/it.yml

```
@@ -1600,7 +1600,7 @@ it:
       subject: Il tuo appello del %{date} è stato respinto
       title: Appello respinto
     backup_ready:
-      explanation: Hai richiesto un backup completo del tuo account %{title}. È pronto per essere scaricato!
+      explanation: Hai richiesto un backup completo del tuo account Mastodon}. È pronto per essere scaricato!
```

There seem to be an unintended trailing `}`
```suggestion
      explanation: Hai richiesto un backup completo del tuo account Mastodon. È pronto per essere scaricato!
```